src/components/Modal/staking-hub/AddAddressToERC1820RegistryModal.tsx

```
@@ -0,0 +1,217 @@
+          onClick={() => { handleCloseModal() }}
+        >
+          <CloseIcon />
+        </SIconButton>
+      </STopBar>
+      <Content>
+        <p>A one time, additional transaction must be done prior to the wrapping.<br />When a Safe wraps xHOPR to wxHOPR, it will fail due to lack of a callback handler. You need to set interface implementer in the{' '}
+          <a
+            href="https://gnosisscan.io/address/0x1820a4b7618bde71dce8cdc73aab6c95905fad24#code"
+            target="_blank"
+            style={{ textDecoration: 'underline' }}
+          >
```

ESLint

[error] Using target="_blank" without rel="noreferrer" (which implies rel="noopener") is a security risk in older browsers: see https://mathiasbynens.github.io/rel-noopener/#recommendations

(react/jsx-no-target-blank)